src/hello-visualization/scatter-plot/app.css

```diff
@@ -0,0 +1,281 @@
+@font-face {
```

Comment:
rename the file to line-chart and remove the css classes not used in this example